application/src/main/java/org/thingsboard/server/service/queue/DefaultTbCoreConsumerService.java

```diff
@@ -195,6 +199,7 @@ public void init() {
         this.consumersExecutor = Executors.newCachedThreadPool(ThingsBoardThreadFactory.forName("tb-core-consumer"));
         this.usageStatsExecutor = Executors.newSingleThreadExecutor(ThingsBoardThreadFactory.forName("tb-core-usage-stats-consumer"));
         this.firmwareStatesExecutor = Executors.newSingleThreadExecutor(ThingsBoardThreadFactory.forName("tb-core-firmware-notifications-consumer"));
+        this.deviceActivityEventsExecutor = MoreExecutors.listeningDecorator(Executors.newSingleThreadExecutor(ThingsBoardThreadFactory.forName("tb-core-device-activity-events-consumer")));
```

Comment:
this executor used only to forward events to device state service. So consider to replace consumer with something else, or remove it at all.

Reply:
Renamed in https://github.com/thingsboard/thingsboard/pull/9030/commits/691a6b5d6a1cc0acd300eca5d9396e296207bf82